jsk_recognition_msgs/msg/Segment.msg

```diff
@@ -0,0 +1,3 @@
+Header header
```

Comment:
I prefer splitting `Segment.msg` and `SegmentStamped.msg`, what do you think about that?